doc/cookbook/source/examples/evaluation/cross_validation_pipeline.rst

```diff
@@ -0,0 +1,34 @@
+============================
+Cross Validation on Pipeline
+============================
+
+In this example, we illustrate how to use cross-validation with :sgclass:`CPipeline`.
```

Comment:
maybe link to some cross-validation cookbook. There is some way to link to notebooks, I think MKL does this

Reply:
ha it is just below, nevermind :)